src/backend/access/bitmap/bitmapattutil.c

```diff
@@ -335,6 +336,11 @@ _bitmap_insert_lov(Relation lovHeap, Relation lovIndex, Datum *datum,
 	result = index_insert(lovIndex, indexDatum, indexNulls,
 					 	  &(tuple->t_self), lovHeap, true);
 
+	SIMPLE_FAULT_INJECTOR("insert_bmlov_before_freeze");
```

Comment:
Why you decided to change usage of fault injectors here comparable to original commit?

Reply:
Because it didn't work with the original.